test_docs.py

```
@@ -76,6 +76,15 @@
     contents = ""
     for line in f.readlines():
         line = line.strip()
+        if line.startswith("git clone"):
+            integration_branch = os.environ.get("INTEGRATION_BRANCH")
+            if integration_branch is not None:
+                words = line.split(" ")
+                default_int_branch = words[3]
```

This is quite weak. Using a regex you can make it more robust and fail explicitly when the expression does not match.